tools/doc/generate.js

```diff
@@ -30,13 +30,13 @@ const fs = require('fs');
 const args = process.argv.slice(2);
 let format = 'json';
 let template = null;
-let inputFile = null;
+let filename = null;
```

Comment:
Using `filename` here is consistent with the name of the variable in other files and allows to pass the `options` object without renaming properties.